packages/request-external-access/test/index.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * External dependencies
+ */
+import PopupMonitor from '@automattic/popup-monitor';
+
+/**
+ * Internal dependencies
+ */
+import requestExternalAccess from '../src';
+
+jest.mock( '@automattic/popup-monitor' );
+
+const popupMonitorMocks = {
+	open: jest.fn(),
+	once: jest.fn(),
+	on: jest.fn(),
+	getScreenCenterSpecs: jest.fn( () => '<screenSpecs>' ),
+};
+
+PopupMonitor.mockImplementation( () => ( { ...popupMonitorMocks } ) );
+
+describe( 'requestExternalAccess', () => {
+	const serviceURL = 'https://foobar.com';
+	const callback = jest.fn();
+
+	beforeEach( () => {
+		requestExternalAccess( serviceURL, callback );
+	} );
+
+	afterEach( () => {
+		Object.values( popupMonitorMocks ).forEach( ( fn ) => fn.mockClear() );
+		callback.mockClear();
+	} );
+
+	afterAll( () => {
+		PopupMonitor.mockReset();
+	} );
+
+	test( 'sets the popup "open" event listener with correct params', () => {
+		expect( popupMonitorMocks.open ).toHaveBeenCalledWith(
+			'https://foobar.com',
```

Comment:
This is an existing 3rd party domain we don't own. Let's replace it in all places in this file with something ours or example.com, which is dedicated for this purpose.

Reply:
Replaced in https://github.com/Automattic/wp-calypso/pull/41990/commits/3fd0e1024d1f2a9db7fe3a0bd11970cc982a81a6.